manifests/plugin.pp

```diff
@@ -22,16 +22,23 @@
   $config_content  = undef,
   $update_url      = undef,
   $plugin_dir      = '/var/lib/jenkins/plugins',
-  $username        = 'jenkins',
-  $group           = 'jenkins',
   $enabled         = true,
   $create_user     = true,
   $source          = undef,
 ) {
   include ::jenkins::params
 
-  $plugin            = "${name}.hpi"
-  $plugin_parent_dir = inline_template('<%= @plugin_dir.split(\'/\')[0..-2].join(\'/\') %>')
+  $plugin     = "${name}.hpi"
+  $homedir    = $::jenkins::params::homedir
+  $jenkinsdir = $::jenkins::params::jenkinsdir
+  $username   = $::jenkins::params::jenkins_user
+  $group      = $::jenkins::params::jenkins_group
```

Comment:
iirc, it's used a bit way down, and since they are differnet on OpenBSD than elsewhere, I did it that way.
Have to review, since after looking at it again, I think now, there might be dragons lurking anyways ;)